### ISSUE TITLE
adding open in option for editing the files in other iOS app

### ISSUE THREAD

**Issue by xu4wang** [tnantoka/edhita#15]
This is a feature request.  I just installed edhita and would like to use it edit the markdown files in my clould app.  It would be great if edhita can expose such a service.

http://stackoverflow.com/questions/3981199/adding-open-in-option-to-ios-app

**Comment by devrando** [comment]
Yes that would be perfect, taking Notes with Editha and saving Seafile/Dropbox/Owncloud etc.

**Comment by julianfairfax** [comment]
@tnantoka Would I be correct in saying that with https://github.com/tnantoka/edhita/commit/4cfb6f382b6e82ffe61daf732ca33b097b05ca2b, it is now possible to use this app to edit files from the files app (like Nextcloud)? And if so, do they have to be opened from the files app or is there an option to open files from the files app in the app?
